bridge/autoload.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-if (getenv('DD_AUTOLOAD_NO_COMPILE') === 'true' || !file_exists(__DIR__ . '/_generated.php')) {
+if (getenv('DD_AUTOLOAD_NO_COMPILE') === 'true') {
```

Comment:
I think that rather than attempt sources, we should just return (if we want to do this check). In any case we are testing real package now (in progress), so I believe this should be removed altogether.